src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const DISCOUNT_FOR_WEEK = 50;
+  const DISCOUNT_FOR_THREE_DAYS = 20;
+
+  let totalCost = days * COST_PER_DAY;
+
+  if (days >= 7) {
+    totalCost -= DISCOUNT_FOR_WEEK;
+  } else if (days >= 3) {
```

Comment:
Same thing with a number 3 here. If you have any questions you can ask in the FE chat